### ISSUE TITLE
UnusedUses bug with complex string interpolation

### ISSUE THREAD

**Issue by wdoug** [slevomat/coding-standard#1252]
The `UnusedUses` rule is giving a false positive for a class that is used in a complex string interpolation. Here is a simplification of my code:

```php
use Namespace\Params;

// lower down in a method in the file:
$variable = "String interpolation: {$this->getParameter(Params::PARAM_NAME)}";
```

I would expect this to have no errors, but instead, `Unuseduses` says that `Namespace\Params` is unused and should be removed.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/add90829a3c1884e140c0a41bcf166f0dc5793e5

**Comment by wdoug** [comment]
@kukulich when do you expect the fix for this to be released?
